LOMS/src/render/uiText.js

```diff
@@ -17,6 +17,16 @@ export default class UIText extends Actor {
             this._sprite.mousedown = (e) => {
                 this._onClick(e);
             }
+            this._sprite.mouseover = (e) => {
+                this._sprite.style = { ...this._style, ...{
+                        stroke: 'white',
+                        strokeThickness: 5,
+                    }
+                };
+            }
```

Comment:
should be this way `this._sprite.style = { ...this._style, 
                        stroke: 'white',
                         strokeThickness: 5
                 };`

Reply:
fixed